proposals/exception-handling/Exceptions.md

```diff
@@ -45,50 +45,40 @@ instruction. Thrown exceptions are handled as follows:
 1. If the call stack is exhausted without any enclosing try blocks, the embedder
    defines how to handle the uncaught exception.
 
-### Event handling
+### Exception handling
 
 This proposal adds exception handling to WebAssembly. Part of this proposal is
 to define a new section to declare exceptions. However, rather than limiting
 this new section to just defining exceptions, it defines a more general format
-that allows the declaration of other forms of events.
+`tag` that allows the declaration of other forms of typed tags in future.
 
-In general, an event handler allows one to process an event generated by a block
-of code. Events suspend the current execution and look for a corresponding event
-handler. If found, the corresponding event handler is run. Some event handlers
-may send values back to the suspended instruction, allowing the originating code
-to resume.
+WebAssembly tags are defined in a new `tag` section of a WebAssembly module. The
+tag section is a list of declared tags associated with the module.
 
-Exceptions are a special case of an event in that they never resume. Similarly,
-a `throw` instruction is the suspending event of an exception. The catch block
-associated with a try block defines how to handle the throw.
-
-WebAssembly events (i.e. exceptions) are defined by a new `event` section of a
-WebAssembly module. The event section is a list of declared events associated
-with the module.
-
-Each event has an `attribute` and a `type`. Currently, the attribute can only
-specify that the event is an exception. In the future, additional attribute
-values may be added when other events are added to WebAssembly.
+Each tag has an `attribute` and a `type`. Currently, the attribute can only
+specify that the tag is for an exception. In the future, additional attribute
+values may be added when other kinds of tags are added to WebAssembly.
 
 To allow for such a future extension possibility, we reserve a byte in the
 binary format of an exception definition, set to 0 to denote an exception
-attribute, but for the moment we won't use the term event in the formal spec.
+attribute.
 
 ### Exceptions
 
-An `exception` is an internal construct in WebAssembly. WebAssembly exceptions
-are defined in the event and import sections of a module.
+An `exception tag` is a value to distinguish different exceptions, while an
+`exception tag index` is a numeric name to refer to an (imported or defined)
+exception tag within a module (see [tag index space](#tag-index-space) for
+details). Exception tags are defined in the tag and import sections of a module.
```

Comment:
Nit (because an import is not a definition in itself):
```suggestion
details). Exception tags are declared in the tag and import sections of a module.
```